staticjinja/staticjinja.py

```diff
@@ -286,27 +286,27 @@ def is_static(self, filename):
         """
         return any(filename.startswith(path) for path in self.staticpaths)
 
-    def is_partial(self, filename):
-        """Check if a file is a partial. Partial files are not rendered,
+    def is_partial(self, template_name):
+        """Check if a template is a partial. Partial templates are not rendered,
         but they are used in rendering templates.
 
-        A file is considered a partial if it or any of its parent directories
-        are prefixed with an ``'_'``.
+        A template is considered a partial if it or any of its parent
+        directories are prefixed with an ``'_'``.
 
-        :param filename: the name of the file to check
+        :param template_name: the name of the template file to check
         """
-        return any((x.startswith("_") for x in filename.split(os.path.sep)))
+        return any((x.startswith("_") for x in template_name.split("/")))
 
-    def is_ignored(self, filename):
-        """Check if a file is an ignored file. Ignored files are neither
-        rendered nor used in rendering templates.
+    def is_ignored(self, template_name):
+        """Check if a template is an ignored template. Ignored templates are
+        neither rendered nor used in rendering templates.
 
-        A file is considered ignored if it or any of its parent directories
+        A template is considered ignored if it or any of its parent directories
         are prefixed with an ``'.'``.
 
-        :param filename: the name of the file to check
+        :param template_name: the name of the template file to check
         """
-        return any((x.startswith(".") for x in filename.split(os.path.sep)))
+        return any((x.startswith(".") for x in template_name.split("/")))
 
     def is_template(self, filename):
```

Comment:
Sorry, this (and the docstring) needs to be `template_name` too :(

Reply:
I have updated docstring to make more sense from it.
```
file -> template
files -> templates
filename -> template_name
```
Can you please help me with your suggestion. 
I am happy to make those changes

Reply:
Sorry that I took so long to get back to you! I found an issue and kept on thinking about how to solve it, but always gave up after 10 minutes and then didn't get back to this until the next day ;)

Originally by this comment I meant using `template_name` as the arg to `is_template()`, but then I noticed that that didn't make much sense. It was like using `truth` as an arg to `is_true()`. This got me going and I read through a bunch more [issues on the Jinja2 repo](https://github.com/pallets/jinja/issues?q=is%3Aissue+template+path) trying to figure out what to call these args. Then I got to worrying that our changes here will break things for existing users, and maybe we do need to keep these args as `filename`. I still haven't figured it out yet, sorry. Was thinking about trying to find some example users of this library to see what they are doing. Will come back to this problem with some actual actionable thoughts. Let me know if you have thoughts. Sorry to be so slow on this!